g2p/apply.py

```diff
@@ -16,7 +16,10 @@ class ApplyG2PModelJob(Job):
     Apply a trained G2P on a word list file
     """
 
-    __sis_hash_exclude__ = {"filter_empty_words": False}
+    __sis_hash_exclude__ = {
+        "filter_empty_words": False,
+        "concurrent": 1,
```

Comment:
I think one does not want to hash this value at all, so I think it is better to add a custom hash function and drop "concurrent" from the list of parameters. There should be other jobs where you can look at how this is typically done.

Reply:
Added a custom hash function that does this now.